Public/Src/Utilities/Configuration/ISandboxConfiguration.cs

```diff
@@ -262,5 +262,12 @@ public interface ISandboxConfiguration
         /// Temporary flag to use tool incremental behavior when preserve outputs is enabled.
         /// </summary>
         bool PreserveOutputsForIncrementalTool { get; }
+
+        /// <summary>
+        /// Flag indicate if change based code coverage is enable.
+        /// This will be set as true when inputChanges commmand line option is presented.
+        /// If the change list contain any removal, it will fall back to false.
+        /// </summary>
+        bool EnableChangeBasedCodeCoverage { get; set; }

```

Comment:
BuildXL doesn't have code coverage. Also, don't mention QTest here. We should not be specific to QTest. 

Name the feature based on its behavior.